samples/client/petstore/python_disallowAdditionalPropertiesIfNotPresent/petstore_api/model/enum_class.py

```diff
@@ -50,6 +50,15 @@ class EnumClass(ModelSimple):
           as additional properties values.
     """
 
+
+    """
+    allowed enum values
+    """
+
+    _ABC = "_abc",
+    -EFG = "-efg",
+    (XYZ) = "(xyz)",
```

Comment:
Well this won't work, maybe we'll need a different solution for special characters.